packages/grafana-data/src/types/navModel.ts

```diff
@@ -17,6 +17,7 @@ export interface NavModelItem {
   showOrgSwitcher?: boolean;
   onClick?: () => void;
   menuItemType?: NavMenuItemType;
+  highlightText?: string;
```

Comment:
I saw @ashharrison90 previous comment about `suffix` not making sense outside of the context of tabs. Maybe name this `tabSuffix` instead? There are other properties of this type that are tab/menu specific. 

The root issue here is that we are using the same model for both tabs and main navigation. But that is hopefully something that we can change in the navigation project.

Reply:
`highlightText` is also used by the main navigation, where it comes from backend, so cannot be a React component. A solution here would be to add a separate type property for tab navigation, but not sure if that's a good idea.

Reply:
Ah, ok! Then I guess it makes sense to leave it as it is. The only thing that feels a bit weird with this solution is that we are trying to be generic in naming of the properties but if you pass a suffix to the Tab component a ProBadge will be added.

If I was using this component and passing a suffix to it I wouldn't expect that behavior. But I guess we need to be pragmatic for now and try to introduce some more flexibility in the navigation models during the navigation project.

Reply:
Yeah, this is quite a tricky issue, on one hand we'd like to make the component easily extensible but on the other I'm not sure we'd want to allow rendering anything without restriction as a suffix. We can revisit this component later if the need for greater extensibility arises and meanwhile I think renaming suffix/tabSuffix props to suffixText/tabSuffixText should at least make it clearer what kind of prop is expected.